### PR TITLE
Clean overriding base class for scroll sync

### PR DESCRIPTION
**Pull request by muxin**

### CODE REVIEW

File: extensions/amp-fx-scroll-sync/0.1/scroll-sync-effect.js [hunk 1]
```diff
@@ -17,6 +17,8 @@
 export class ScrollSyncEffect {
   constructor(element) {
     this.element = element;
+    this.scrollMin_ = null;
```
Comment (jridgewell) [inline extensions/amp-fx-scroll-sync/0.1/scroll-sync-effect.js]: Let's use `0` so they're always `number`s.

File: extensions/amp-fx-scroll-sync/0.1/scroll-sync-effect.js [hunk 2]
```diff
@@ -17,6 +17,8 @@
 export class ScrollSyncEffect {
   constructor(element) {
     this.element = element;
```
Comment (jridgewell) [inline extensions/amp-fx-scroll-sync/0.1/scroll-sync-effect.js]: Please typehint everything here.

**Comment by muxin** [comment]
PTAL

**Comment by jridgewell** [comment]
LGTM.

**Pull request merged**